frontend/src/lib/components/icons.scss

```
@@ -22,3 +21,19 @@
         }
     }
 }
+
+.icon-count-bubble {
+    position: absolute;
+    width: 16px;
+    height: 16px;
+    line-height: 16px;
+    top: -0.3rem;
+    right: -0.3rem;
```

The count is a bit further from the center of the icon in the design as the icon is more easily recognizable then.
```suggestion
    top: -0.375rem;
    right: -0.375rem;
```